### ISSUE TITLE
1822 incorrect purchase price listed

### ISSUE THREAD

**Issue by ChrisTheCat** [tobymao/18xx#4398]
https://18xx.games/game/31523?action=55

log says:

[09:45]ChrisShaffer pars 24 at £50
[09:45]ChrisShaffer buys a 100% share of 24 from the IPO for £100
[09:45]ChrisShaffer becomes the president of 24
[09:45]24 floats

However, I won the bid for 115, not 100.

**Comment by magnusfalt** [comment]
Yes the game does this correct. The log doesnt say how much money that was taken from you though. Phase 1 price is always 100 what ever you bid and rest put into the bank.

You had before all passed: £525.
Winning bid on MINOR: 24: £115.
Winning bid on CONCESSION: LBSCR: £110.

After stockround you have: £300.
You collects £10 from CONCESSION: LBSCR in revenue.

You should now have: £310, and that is what you have at this point in the game.

**Comment by ChrisTheCat** [comment]
I was not reporting that the game had done something wrong with the math. Rather, I was reporting that the log text is incorrect and should be edited to more correctly describe what happened in the game.

The amount deducted from the player is correct as you note, but it is not correct to say that the player bought the minor for 100.

It's not "bid 115, discard 15, buy minor for 100" but rather "buy minor for 115, then set par price at 50 with 100 treasury."

Rules say "players take the items for which they have bid the most, paying the amount bid." If the winning bid is 115, it's not correct to say that the player "paid 100." The log should say that the player paid 115.

**Comment by ChrisTheCat** [comment]
Here's one potential log text:

[09:45]ChrisShaffer buys a 100% share of 24 from the IPO for £115
[09:45]ChrisShaffer becomes the president of 24
[09:45]Chris Shaffer pars 24 at £50
[09:45]24 floats with a starting treasury of £100

And if the game were in phase 2, it would look like this:

[09:45]ChrisShaffer buys a 100% share of 24 from the IPO for £115
[09:45]ChrisShaffer becomes the president of 24
[09:45]Chris Shaffer pars 24 at £55
[09:45]24 floats with a starting treasury of £110

**Comment by magnusfalt** [comment]
In the linked PR ive added a line that says how much money the player have paided to the bank.
